src/js/component/Controller.js

```diff
@@ -16,51 +16,52 @@ class Controller {
   constructor (view, model) {
     this.view = view
     this.model = model
+    this.channels = {}
+    this.articles = []
   }
 
-  append(parent, el) {
-    return parent.appendChild(el)
+  setArticles(articles) {
+    this.articles = articles  
+    this.view.disableSpinner()
+    return this.articles
   }
 
-  createNode(element) {
-    return document.createElement(element)
-  }    
-
   get(source) {
     this.view.enableSpinner()   
-    return proxiedApi.get(`${BASE_URL}${source}&apiKey=${API_KEY}`)
-  }
+    return proxiedNewsApi.get(source)
+  }  
 
-  renderItem(news) {
-    let li = this.view.createNode('li'), 
-        img = this.view.createNode('img'),
-        div = this.view.createNode('div'),
-        p = this.view.createNode('p'),
-        h4 = this.view.createNode('h4'),
-        strong = this.view.createNode('strong'); 
+  render() {
+    this.view.ul.innerHTML = ''
+    this.articles.map(item => this.view.renderItem(item))
+  }
+  
+  fetchChannel(source) {
```

Comment:
Do we really need to have a get method inside of controller? Can we just use proxiedApi?